llvm-hs/src/LLVM/Internal/FFI/TargetC.cpp

```diff
@@ -322,6 +334,20 @@ unsigned LLVM_Hs_GetTargetOptionFlag(TargetOptions *to,
     }
 }
 
+unsigned LLVM_Hs_GetMCTargetOptionFlag(MCTargetOptions *to,
+                                       LLVM_Hs_MCTargetOptionFlag f) {
+    switch (f) {
+#define ENUM_CASE(op)                                                          \
+    case LLVM_Hs_MCTargetOptionFlag_##op:                                      \
+        return to->op;
+        LLVM_HS_FOR_EACH_MC_TARGET_OPTION_FLAG(ENUM_CASE)
+#undef ENUM_CASE
+    default:
+        assert(false && "Unknown machine code target option flag");
```

Comment:
Please use [reportFatalError](https://github.com/llvm-hs/llvm-hs/blob/llvm-5/llvm-hs/src/LLVM/Internal/FFI/ErrorHandling.hpp#L5) for this.

Reply:
I can do this, but note that this file contains 3 other asserts and zero calls to reportFatalError. Maybe those should be changed too.

Reply:
Yeah they definitely should be changed. I just haven’t gotten around to it yet.

Reply:
I can change them all if you don't mind.

Reply:
That would be great!